tests/base.go

```diff
@@ -73,78 +64,80 @@ func RemoteRun(user string, addr string, cmd string) (string, error) {
 	// Connect
 	client, err := ssh.Dial("tcp", net.JoinHostPort(addr, "22"), config)
 	if err != nil {
-		return "", err
+		return "", errors.Wrapf(err, "error starting ssh connection ")
 	}
 	// Create a session. It is one session per command.
 	session, err := client.NewSession()
 	if err != nil {
-		return "", err
+		return "", errors.Wrapf(err, "error creating new session")
 	}
 	defer session.Close()
 	var b bytes.Buffer  // import "bytes"
 	session.Stdout = &b // get output
 	err = session.Run(cmd)
-	return b.String(), err
+	if err != nil {
+		return "", errors.Wrapf(err, "error excuting command on remote")
+	}
+	return b.String(), nil
 }
 
-func VerifyIPs(wgConfig string, verifyIPs []string) bool {
-	UpWg(wgConfig)
+// TODO: investigate if this is needed
+func VerifyIPs(wgConfig string, verifyIPs []string, sk string) error {
 
 	for i := 0; i < len(verifyIPs); i++ {
-		out, _ := exec.Command("ping", verifyIPs[i], "-c 5", "-i 3", "-w 10").Output()
+		out, err := exec.Command("ping", verifyIPs[i], "-c 5", "-i 3", "-w 10").Output()
 		if strings.Contains(string(out), "Destination Host Unreachable") {
-			return false
+			return errors.Wrapf(err, "error host unreachable")
 		}
 	}
-
 	for i := 0; i < len(verifyIPs); i++ {
```

Comment:
you aren't changing anything in verifyIPs, you can use for loop with value semantic, using the range